.github/workflows/common-backend.yml

```diff
@@ -0,0 +1,45 @@
+name: Reusable backend workflow
+
+on:
+  workflow_call:
+    inputs:
+      app-name:
+        description: "Application name"
+        required: true
+        type: string
+
+env:
+  BACKEND_DIR: ./backend
+
+jobs:
+  common-backend:
+    if: contains(github.event.head_commit.message, ${{ inputs.app-name }})
```

Comment:
Onko tää `contains` case sensitive? Esim. AKT:n osalta vertailtiin aiemmin merkkijonoon "AKT", mutta nyt inputeista syötetty app-name on lower-case "akt".

Reply:
contains ei ole case sensitive

Reply:
voidaan kuintekin antaa inputit CAPITAL CASE:lla. Tulevaisuudessa tilanne voi muuttua.

Reply:
Jees, niinhän tuolla dokkareissa sanotaan ettei casella ole väliä. Saa olla siis näin.